local_deploy/encode_query_aio.py

```diff
@@ -0,0 +1,128 @@
+import ast
+import asyncio
+import base64
+from io import BytesIO
+import json
+import os
+import time
+import aiohttp
+import numpy as np
+from square_auth.client_credentials import ClientCredentials
+import requests
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def get_token() -> str:
```

Comment:
remove function and use a global ClientCredentials object

Reply:
Done in [bc21930](https://github.com/UKP-SQuARE/square-core/pull/354/commits/bc21930a76e9cf645ed7468ddb4f551615004ee9)